listings/admin.py

```diff
@@ -2,9 +2,35 @@
 
 from .models import Region, City, GatheringCenter, Resource
 
-
 admin.site.register(Region)
 admin.site.register(City)
-admin.site.register(GatheringCenter)
-admin.site.register(Resource)
+
+
+class PublishMixin(object):
+    actions = ('publish', 'unpublish')
+
+    def publish(self, request, queryset):
+        queryset.update(published=True)
+
+    def unpublish(self, request, queryset):
+        queryset.update(published=False)
+
+
+class GatheringCenterAdmin(PublishMixin, admin.ModelAdmin):
+    list_filter = ('published', 'city', 'created')
+    list_editable = ('published', )
+    list_display = ('location_name', 'created', 'published', 'author', 'city')
+    raw_id_fields = ('author', )
+
+admin.site.register(GatheringCenter, GatheringCenterAdmin)
+
+
+class ResourceAdmin(PublishMixin, admin.ModelAdmin):
+    list_filter = ('published', 'created')
```

Comment:
instead of including a `DatetimeField` (`created`) in `list_filter`, I would set `date_hierarchy = 'created'` (https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#django.contrib.admin.ModelAdmin.date_hierarchy). Same for `GatheringCenter`